content/news/2020-02-28.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Work on OpenADx Whitepaper started"
+date: 2020-02-28T17:09:06+00:00
+hide_page_title: "false"
+categories: ["news"]
+#header_wrapper_class: ""
+#seo_title: ""
+#headline: ""
+#subtitle: ""
+#tagline: ""
+#links: []
+---
+
+We started to work on an OpenADx Whitepaper! 
+If you would like to participate and work with us, please let us know! 
+You can contact us via our mailing list: [openadx@eclipse.org](openadx@eclipse.org)
```

Comment:
this won't open an email client, if you want it to open a client, you need to start the link with `mailto:`